packages/playground/css/main.js

```diff
@@ -80,7 +80,9 @@ text('.inlined-code', inlined)
 
 // glob
 const glob = import.meta.glob('./glob-import/*.css')
-Promise.all(Object.keys(glob).map((key) => glob[key]())).then((res) => {
+Promise.all(
+  Object.keys(glob).map((key) => glob[key]().then((i) => i.default))
+).then((res) => {
```

Comment:
How was this working before? 🤔 
Your change looks correct here.

Reply:
The tests check if it "contains". Either with/without this change the test would pass, this is more like a by-product for debugging when I test it. Since it doesn't really affect anything, I am fine with keeping it or reverting it.

Reply:
No, I think the change is good.